frontend/src/metabase-types/api/settings.ts

```diff
@@ -32,6 +32,11 @@ export type LoadingMessage =
   | "running-query"
   | "loading-results";
 
+export type TokenStatus = {
+  status?: "unpaid" | "past-due" | string;
+  [index: string]: unknown;
+};
```

Comment:
This is basically just for documentation, since we want the store's API response to be able to evolve without breaking previous versions of metabase.

Reply:
Please remove `[index: ...]` part. It won't break if there would be additional properties, but currently this would break typechecking.

Reply:
Thanks, I didn't know that, but it makes sense. I had imagined this type-checking was similar to run-time validations in clojure, but actually they're absent at run-time.